react/features/base/media/components/web/VideoTrack.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import AbstractVideoTrack from '../AbstractVideoTrack';
+
+/**
+ * Component that renders a video element for a passed in video track.
+ *
+ * @extends AbstractVideoTrack
+ */
+class VideoTrack extends AbstractVideoTrack {
+    static defaultProps = {
+        /**
+         * Dispatch an action when the video starts playing.
+         */
+        triggerOnPlayingUpdate: true
+    };
+
+    static propTypes = {
```

Comment:
Sure, I moved over triggerOnPlayingUpdate with a default value of true. I don't think id would be of any interest... The only interest web has is for torture tests.